### DISCUSSION TITLE
Angular and clarity version compatibility

### DISCUSSION THREAD

**Discussion by swathishetty** [vmware-clarity/ng-clarity#859] [Q&A] [answered]
Can we use higher version of clarity with lower version of angular ?
Can i use clarity 4 with Angular 9 ?

**Comment by kevinbuhmann** [comment, marked as answer]
Please see the support policies. https://clarity.design/documentation/support

**Reply by jerone** [reply]
@kevinbuhmann What is the difference between the "Status" column in "Support Policy & Schedule" and "Clarity Design System and Angular Compatibility" tables?

Because both have the Clarity version, but in the second table v15 is marked LTS, while in the first table it's marked as "Actively Supported".
![image](https://github.com/vmware-clarity/ng-clarity/assets/55841/f445bb54-12fd-4664-916e-3182d0167696)

_Minor nitpic: call the first column in the first table also "Clarity Version" for consistency with the second table._

**Reply by kevinbuhmann** [reply]
That second table used to list the Angular support status. We changed it though to reduce confusion and because the range of compatible Angular versions will have varying support statuses. Both status columns now state the Clarity support status.
